src/workloads/encrypted/qe-range-age-100-0.yml

```diff
@@ -0,0 +1,1036 @@
+      InitialStates:
+        - State: Query
+          Weight: 1
+        - State: Update
+          Weight: 0
+      States:
+        - Name: Query
+          Operations:
+          - OperationName: find
+            OperationMetricsName: range_query
+            OperationCommand:
+              Filter: { age_hospitals: {
+                  $gte: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/queries/rc_age_t_min.txt", "sequential": true, "startFromLine": 18750}}}},
+                  $lte: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/queries/rc_age_t_max.txt", "sequential": true, "startFromLine": 18750}}}}
+                }
+              }
+          Transitions: *transitions
+        - Name: Update
+          Operations:
+          - OperationName: updateOne
+            OperationMetricsName: update
+            OperationCommand:
+              Filter: { age_hospitals: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/data/rc_ages_up.txt"}}}}}
+              Update: {$set: { age_hospitals: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/data/rc_ages.txt"} } } } } }
+          Transitions: *transitions
+
+  - Name: FSMActor_Thread4
+    Type: CrudActor
+    Threads: 1
+    Database: *encrypted_db
+    ClientName: EncryptedPool
+    
+    Phases:
+    - *nop
+    - *nop
+    - Repeat: 6250
+      Collection: *encrypted_coll
+      MetricsName: "FSM"
+      InitialStates:
+        - State: Query
+          Weight: 1
+        - State: Update
+          Weight: 0
+      States:
+        - Name: Query
+          Operations:
+          - OperationName: find
+            OperationMetricsName: range_query
+            OperationCommand:
+              Filter: { age_hospitals: {
+                  $gte: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/queries/rc_age_t_min.txt", "sequential": true, "startFromLine": 25000}}}},
+                  $lte: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/queries/rc_age_t_max.txt", "sequential": true, "startFromLine": 25000}}}}
+                }
+              }
+          Transitions: *transitions
+        - Name: Update
+          Operations:
+          - OperationName: updateOne
+            OperationMetricsName: update
+            OperationCommand:
+              Filter: { age_hospitals: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/data/rc_ages_up.txt"}}}}}
+              Update: {$set: { age_hospitals: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/genny/src/workloads/contrib/qe_range_testing/data/rc_ages.txt"} } } } } }
+          Transitions: *transitions
+
+  - Name: FSMActor_Thread5
```

Comment:
These actors could also be templated as startFromLine seems to be the only variation.

Reply:
Templated here as well.